code/modules/organs/organ.dm

```diff
@@ -134,7 +172,7 @@ var/list/organ_cache = list()
 	handle_organ_proc_special()
 
 	//Process infections
-	if(robotic >= ORGAN_ROBOT || (owner && owner.species && (owner.species.flags & IS_PLANT || (owner.species.flags & NO_INFECT))))
+	if(robotic >= ORGAN_ROBOT || (istype(owner) && (owner && owner.species && (owner.species.flags & IS_PLANT || (owner.species.flags & NO_INFECT)))))
```

Comment:
`istype(owner)` enforces `owner`, so the former can replace the latter, rather than being needed in addition to